src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/vlboptionsfinder/OptionsGenerator.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.vlboptionsfinder
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.format.annotation.DateTimeFormat
+import org.springframework.stereotype.Component
+import java.time.Duration
+import java.time.LocalTime
+
+@Component
+class OptionsGenerator(
+  @DateTimeFormat(iso = DateTimeFormat.ISO.TIME)
+  @Value("\${video-link-booking.day-start}")
```

Comment:
I quite like using `@ConfigurationProperties` at the moment.  If you load them into a bean then you can make it a `@SpyBean` in integration tests meaning you don't need to start a new Spring context when testing different values in the properties.  Or maybe we just write too many integration tests on our projects :smiley: